chart/values.yaml

```diff
@@ -1,7 +1,7 @@
 replicaCount: 1
 
 image:
-  name: thomasr/smi-metrics:v1alpha1-0.1
+  name: deislabs/smi-metrics:VERSION
```

Comment:
Shouldn’t this be chart version?

Reply:
`CHART_VERSION` dosent include the `v` right. Docker images should have the version with `v` I guess?